rafx-renderer/src/render_frame_job.rs

```diff
@@ -1,27 +1,25 @@
 use super::Renderer;
-use super::RendererPlugin;
+use crate::{RenderFeaturePlugin, RendererThreadPool};
+use fnv::FnvBuildHasher;
 use rafx_api::{RafxCommandBuffer, RafxDeviceContext, RafxQueue};
 use rafx_api::{RafxPresentableFrame, RafxResult};
 use rafx_framework::graph::PreparedRenderGraph;
-use rafx_framework::nodes::{
-    FramePacket, PrepareJobSet, RenderJobPrepareContext, RenderRegistry, RenderView,
-};
+use rafx_framework::render_features::render_features_prelude::*;
 use rafx_framework::{DynCommandBuffer, RenderResources, ResourceContext};
 use std::sync::Arc;
 
 pub struct RenderFrameJobResult;
 
 pub struct RenderFrameJob {
```

Comment:
docs for what this does

Reply:
Added a `TODO(dvd): DOCUMENT` to track that.